context/app/static/js/shared-styles/sections/Description/style.js

```diff
@@ -4,7 +4,7 @@ import Paper from '@material-ui/core/Paper';
 
 const StyledPaper = styled(Paper)`
   display: flex;
-  padding: 30px 40px 30px 40px;
+  padding: ${(props) => props?.$padding || '30px 40px'};
```

Comment:
ok... This is where that padding attribute above gets used. Maybe to keep the CSS details out of the JSX, instead of only having `StyledPaper`, make `BigPaddingPaper` and `SmallPaddingPaper`?